package.json

```diff
@@ -187,7 +187,7 @@
     "script": "babel-node --extensions .js,.ts $1",
     "sequelize": "babel-node --extensions .js,.ts -- ./node_modules/.bin/sequelize --config config/sequelize-cli.js",
     "start": "node ./dist/index.js",
-    "test": "TZ=UTC nyc --reporter=lcov mocha",
+    "test": "TZ=UTC nyc --reporter=lcov mocha --config test/.mocharc.json",
     "test:clean": "rm -rf .nyc_output coverage",
     "test:coverage": "codecov",
     "test:graphql": "TZ=UTC mocha test/server/graphql",
```

Comment:
Looks like the config is missing in all the other commands.

In the end, this might be verbose and unpractical to always require to pass it on the CLI.

Let's settle on the root file approach (changed my mind compared to the original specification).